src/charm.py

```diff
@@ -127,14 +134,28 @@ def _reconcile(self):
         remote_write.reload_alerts()
         self._add_remote_write(remote_write.endpoints)
 
+        # Enable forwarding telemetry with GrafanaCloudIntegrator
+        cloud_integrator = GrafanaCloudConfigRequirer(self, relation_name="cloud-config")
+        # We're intentionally not getting the CA cert from Grafana Cloud Integrator;
+        # we decided that we should only get certs from receive-ca-cert.
+        self._add_cloud_integrator(
+            credentials=cloud_integrator.credentials,
+            prometheus_url=cloud_integrator.prometheus_url
+            if cloud_integrator.prometheus_ready
+            else None,
+            loki_url=cloud_integrator.loki_url if cloud_integrator.loki_ready else None,
+        )
+
```

Comment:
I would extract these lines to a private method to make the `_reconcile` method shorter and cleaner